src/main/java/appeng/core/sync/AppEngPacket.java

```diff
@@ -19,60 +19,52 @@
 package appeng.core.sync;
 
 
-import io.netty.buffer.ByteBuf;
+import cpw.mods.fml.common.network.simpleimpl.IMessage;
 
-import net.minecraft.entity.player.EntityPlayer;
 
-import cpw.mods.fml.common.network.internal.FMLProxyPacket;
-
-import appeng.core.AEConfig;
-import appeng.core.AELog;
-import appeng.core.features.AEFeature;
-import appeng.core.sync.network.INetworkInfo;
-import appeng.core.sync.network.NetworkHandler;
-
-
-public abstract class AppEngPacket
+public interface AppEngPacket extends IMessage
 {
 
-	AppEngPacketHandlerBase.PacketTypes id;
-	private ByteBuf p;
-
-	public void serverPacketData( INetworkInfo manager, AppEngPacket packet, EntityPlayer player )
-	{
-		throw new UnsupportedOperationException( "This packet ( " + this.getPacketID() + " does not implement a server side handler." );
-	}
-
-	public final int getPacketID()
-	{
-		return AppEngPacketHandlerBase.PacketTypes.getID( this.getClass() ).ordinal();
-	}
-
-	public void clientPacketData( INetworkInfo network, AppEngPacket packet, EntityPlayer player )
-	{
-		throw new UnsupportedOperationException( "This packet ( " + this.getPacketID() + " does not implement a client side handler." );
-	}
-
-	protected void configureWrite( ByteBuf data )
-	{
-		data.capacity( data.readableBytes() );
-		this.p = data;
-	}
-
-	public FMLProxyPacket getProxy()
-	{
-		if( this.p.array().length > 2 * 1024 * 1024 ) // 2k walking room :)
-		{
-			throw new IllegalArgumentException( "Sorry AE2 made a " + this.p.array().length + " byte packet by accident!" );
-		}
-
-		FMLProxyPacket pp = new FMLProxyPacket( this.p, NetworkHandler.instance.getChannel() );
-
-		if( AEConfig.instance.isFeatureEnabled( AEFeature.PacketLogging ) )
-		{
-			AELog.info( this.getClass().getName() + " : " + pp.payload().readableBytes() );
-		}
-
-		return pp;
-	}
+	// AppEngPacketHandlerBase.PacketTypes id;
+	// private ByteBuf p;
+	//
+	// public void serverPacketData( INetworkInfo manager, AppEngPacket packet, EntityPlayer player )
+	// {
+	// throw new UnsupportedOperationException( "This packet ( " + this.getPacketID() +
+	// " does not implement a server side handler." );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)